slack.md

```diff
@@ -1,15 +1,31 @@
 ---
 title: Slack
+sidebar_link: true
+sidebar_sort_order: 600
 ---
 
-Boston Python has a Slack workspace. Please join us!
+Boston Python has a Slack workspace for online discussions.
 
-First, [sign up for the Boston Python Slack](https://slack.bostonpython.com).
+[Sign up to join us!](https://slack.bostonpython.com) As always, we expect all members to abide by our [code of conduct](code-of-conduct.md).
 
-Once in, there are a number of channels, depending on what you are looking for:
+Once you're in, there are a number of channels, depending on what you are looking for:
 
-- **#general** is the default channel. When in doubt, ask here.
-- **#organizing** is for talking about how the group works, and making it work better.
-- **#jobs** is for job postings that meet our [job posting requirements](jobs.md).
-- **#beginners** is for beginners looking for more in-depth help.
-- **#events** is for announcing events.
+- **#general**: Questions/ideas/links of general interest to the community.
+- **#events**: Announcements/discussion of events that might be interesting to the group.
+- **#jobs**: For job postings that meet our [requirements](jobs.md), or folks looking for work.
```

Comment:
I hadn't thought about "looking for work"... We'll have to keep an eye on it getting unproductively noisy.